### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### Changes made

- Created method `current_user` in ApplicationControlller
- Created form to  Post on behalf of the current_user 
- Created form to create a comment on behalf of the current_user
- Added a like button to allow users to like posts

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = @user.comments.new(comment_params)
+    @comment.user_id = @user.id
+    @comment.post_id = params[:post_id]
+    @comment.save
+    redirect_to user_post_path(@user.id, @post)
+  end
+
```
Comment (AnselemOdims) [inline app/controllers/comments_controller.rb]: - [ ] Please kindly include a `new` controller method to the `CommentsController`.

File: config/routes.rb [hunk 2]
```diff
@@ -8,6 +8,10 @@
   root "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
+      resources :comments, only: [:create]
```
Comment (AnselemOdims) [inline config/routes.rb]: - [ ] Please kindly include a `new` route to be handled by the `new` method in the `CommentsController`

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @kemigabocatherine , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bimbolabuari — COMMENTED** [review]
Hello @kemigabocatherine 👋,

_This is an additional review as a result of the second opinion request._

Yes, the CR was trying to improve your codebase,The new method lets Rails know that you will create a new object.

As this was the only reason that your project was not approved, Kindly make the changes and submit for approval.

Keep rocking 🚀!

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Cathy,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- View templates and layout properly implemented

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**